### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by yonas** [GabAlpha/basilk#7]
Even a few would be good.

**Comment by GabAlpha** [comment]
I agree, which ones were you thinking of? I would propose those for writing/reading tasks and projects. I think a temporary JSON file should be created and then deleted after the tests are finished

**Comment by yonas** [comment]
Those sound good.

**Comment by yonas** [comment]
Feel free to use my template repo https://github.com/yonasBSD/rust-ci-github-actions-workflow
